aztk/models.py

```diff
@@ -23,26 +50,35 @@ def __init__(self, name: str = None, script: str = None, run_on=None):
         self.run_on = run_on
 
 
-class UserConfiguration:
+class UserConfiguration(ConfigurationBase):
```

Comment:
This should have a subclass in the `aztk.spark.models` package so people don't have to import `aztk.models` and `aztk.spark.models`.